saleor/graphql/order/bulk_mutations/order_bulk_create.py

```diff
@@ -0,0 +1,1064 @@
+            for data in metadata:
+                order_line.tax_class_metadata.update({data["key"]: data["value"]})
+        if private_metadata := order_line_input.get("tax_class_private_metadata"):
+            for data in private_metadata:
+                order_line.tax_class_private_metadata.update(
+                    {data["key"]: data["value"]}
+                )
+        return order_line
+
+    @classmethod
+    def create_single_order(
+        cls, order_input, object_storage: Dict[str, Any]
+    ) -> OrderWithErrors:
+        order = OrderWithErrors(
+            order=None,
+            order_errors=[],
+            lines=[],
+            lines_errors=[],
+            notes=[],
+            notes_errors=[],
+        )
+        cls.validate_order_input(order_input, order.order_errors)
+        order_instance = Order()
+
+        # get order related instances
+        instances = cls.get_instances_related_to_order(
+            order_input=order_input,
+            errors=order.order_errors,
+            object_storage=object_storage,
+        )
+        delivery_input = order_input["delivery_method"]
+        delivery_method = cls.get_delivery_method(
+            input=delivery_input,
+            errors=order.order_errors,
+            object_storage=object_storage,
+        )
+        if not instances or not delivery_method:
+            return order
+
+        # create lines
+        order_lines_input = order_input["lines"]
+        for order_line_input in order_lines_input:
+            if order_line := cls.create_single_order_line(
+                order_line_input,
+                order_instance,
+                object_storage,
+                order_input,
+                order.lines_errors,
+            ):
+                order.lines.append(order_line)
+            else:
+                order.is_each_line_created = False
+
+        if not order.is_each_line_created:
+            order.order_errors.append(
+                OrderBulkError(
+                    message="At least one order line can't be created.",
+                    field="lines",
+                    code=OrderBulkCreateErrorCode.ORDER_LINE_ERROR,
+                )
+            )
+            return order
+        # TODO check if multiple order lines contains the same variant (fulfillments)
+
+        # calculate order amounts
+        order_amounts = cls.make_order_calculations(
+            delivery_method,
+            order.lines,
+            instances.channel,
+            delivery_input,
+            object_storage,
+        )
+
+        # create notes
+        if notes_input := order_input.get("notes"):
+            for note_input in notes_input:
+                if note := cls.create_single_note(
+                    note_input, order_instance, object_storage, order.notes_errors
+                ):
+                    order.notes.append(note)
+
+        # order.number = order_input.get("number")
```

Comment:
What about this comment?

Reply:
For the need of bulk import, the number field needs to be altered into string, so I want to handle it in separate PR